pb-crm-api/src/db/entities/user.entity.ts

```diff
@@ -1,19 +1,18 @@
 import {
-  BeforeInsert,
-  Column,
-  CreateDateColumn,
-  Entity,
-  JoinColumn,
-  OneToOne,
-  PrimaryGeneratedColumn,
-  UpdateDateColumn,
+    BeforeInsert,
+    Column,
+    CreateDateColumn,
+    Entity,
+    JoinColumn,
+    OneToOne,
+    PrimaryGeneratedColumn,
+    UpdateDateColumn,
 } from 'typeorm';
 import {UserTypeEntity} from "./user-type.entity";
 import {ContactEntity} from "./contact.entity";
-import {UserModel} from '@hiddentemple/api-interfaces';
-import {Length, MaxLength, IsEmail} from "class-validator";
+import {IsEmail, Length, MaxLength} from "class-validator";
 import * as bcrypt from 'bcrypt';
-
+import {UserModel} from "@hiddentemple/api-interfaces/dist/user/models/user.model";
```

Comment:
weird

Reply:
Yeah this is wrong, I forgot to do a barrel import for users